features/product/viewing_product_in_admin_panel/viewing_details_of_product_with_variants.feature

```diff
@@ -17,13 +17,13 @@ Feature: Viewing details of a product with variants
         And I am logged in as an administrator
         And I am browsing products
 
-    @ui
+    @ui @no-api
```

Comment:
Why all of these scenarios are marked as no-api? We can probably check the proper information on GET endpoint of variant/product

Reply:
We most likely could, but pretty much all of them (if not all) are already covered in different scenarios. Don't know if duplicating them is worth it.

Reply:
Added where applicable